name_data/labeled/person_labeled.xml

```diff
@@ -3995,4 +3995,93 @@
+  <Name><PrefixOther>Dr.</PrefixOther> <GivenName>SOK</GivenName> <GivenName>LI</GivenName> <GivenName>ANN</GivenName> <Surname>KUO</Surname></Name>
+  <Name><GivenName>SVITLANA</GivenName> <MiddleInitial>J</MiddleInitial> <Surname>MANDZY</Surname></Name>
+  <Name><PrefixMarital>Miss</PrefixMarital> <GivenName>URVI</GivenName> <MiddleName>PRADEEP</MiddleName> <Surname>VYAS</Surname></Name>
+  <Name><PrefixMarital>Ms</PrefixMarital> <GivenName>NEVEDITA</GivenName> <MiddleInitial>S</MiddleInitial> <Surname>SARASWAT</Surname> <SuffixOther>MD</SuffixOther></Name>
+  <Name><PrefixMarital>Miss</PrefixMarital> <GivenName>JEWELL</GivenName> <MiddleName>ANN</MiddleName> <Surname>CRAWFORD</Surname></Name>
+  <Name><PrefixOther>Dr</PrefixOther> <And>&amp;</And> <PrefixMarital>Mrs</PrefixMarital> <GivenName>RUSKIN</GivenName> <Surname>LINES</Surname></Name>
+  <Name><PrefixMarital>Miss</PrefixMarital> <GivenName>JENNA</GivenName> <Surname>LAMMERS</Surname> <Surname>MCCREE</Surname></Name>
+  <Name><GivenName>HINA</GivenName> <Surname>AAMIR</Surname></Name>
+  <Name><PrefixMarital>Miss</PrefixMarital> <GivenName>MARIETTA</GivenName> <MiddleInitial>F</MiddleInitial> <Surname>ABALOS</Surname> <Surname>GALITO</Surname></Name>
+  <Name><PrefixMarital>Miss</PrefixMarital> <GivenName>CHRISTINA</GivenName> <Surname>MERCY</Surname> <Surname>KEMP</Surname></Name>
+  <Name><GivenName>SUNITA</GivenName> <Surname>REDDY</Surname></Name>
+  <Name><GivenName>CHUN</GivenName> <GivenName>HUNG</GivenName> <Surname>HSUIH</Surname></Name>
+  <Name><GivenName>SHANTISE</GivenName> <Surname>STORY</Surname></Name>
+  <Name><PrefixMarital>Miss</PrefixMarital> <GivenName>LAURA</GivenName> <MiddleName>ANN</MiddleName> <Surname>LA</Surname> <Surname>SELL</Surname></Name>
+  <Name><PrefixOther>Dr.</PrefixOther> <GivenName>FADI</GivenName> <Surname>AL</Surname> <Surname>BILBEISI</Surname></Name>
+  <Name><PrefixMarital>Miss</PrefixMarital> <GivenName>TRACEY</GivenName> <MiddleName>MARIE</MiddleName> <MiddleName>MIRANDA</MiddleName> <Surname>RACO</Surname></Name>
+  <Name><GivenName>EDWARD</GivenName> <MiddleName>JAY</MiddleName> <Surname>POLLYEA</Surname></Name>
+  <Name><PrefixOther>Dr.</PrefixOther> <And>and</And> <PrefixMarital>Mrs.</PrefixMarital> <FirstInitial>E</FirstInitial> <GivenName>WILLIAM</GivenName> <MiddleInitial>W</MiddleInitial> <Surname>JOHNSON</Surname></Name>
+  <Name><PrefixMarital>Mr.</PrefixMarital> <GivenName>A</GivenName> <GivenName>Y</GivenName> <GivenName>M</GivenName> <GivenName>MAKSUDUL</GivenName> <Surname>CHOWDHURY</Surname></Name>
+  <Name><GivenName>CHALERMCHAI</GivenName> <Surname>PUNYA</Surname></Name>
+  <Name><GivenName>BING</GivenName> <MiddleName>WANG</MiddleName> <Surname>GALE</Surname></Name>
+  <Name><GivenName>ALI</GivenName> <MiddleInitial>A</MiddleInitial> <Surname>AL</Surname> <Surname>MUDAMGHA</Surname></Name>
+  <Name><PrefixMarital>Miss</PrefixMarital> <GivenName>DYANN</GivenName> <Surname>HOLLOWAY</Surname> <Surname>DALEY</Surname></Name>
```

Comment:
Is `HOLLOWAY` a `MiddleName` as well?

Reply:
Yes, according to the original CMS Physicians data set HOLLOWAY here is a middle name. However, it looks more like a last name, so it seems the original data set inconsistently records the names of females with two last names written without a hyphen.

I will change this in the XML